src/plugins/three/DebugTilesPlugin.js

```diff
@@ -800,6 +813,67 @@ export class DebugTilesPlugin {
 
 	}
 
+	_updateMaterial( scene ) {
+
+		// update the materials for debug rendering
+		const { colorMode, unlit } = this;
+		scene.traverse( c => {
+
+			if ( ! c.material ) {
+
+				return;
+
+			}
+
+			const currMaterial = c.material;
+			const originalMaterial = c[ ORIGINAL_MATERIAL ];
+
+			// dispose the previous material
+			if ( currMaterial !== originalMaterial ) {
+
+				currMaterial.dispose();
+
+			}
+
+			// assign the new material
+			if ( colorMode !== NONE || unlit ) {
+
+				if ( c.isPoints ) {
+
+					const pointsMaterial = new PointsMaterial();
+					pointsMaterial.size = originalMaterial.size;
+					pointsMaterial.sizeAttenuation = originalMaterial.sizeAttenuation;
+					c.material = pointsMaterial;
+
+				} else if ( unlit ) {
+
+					c.material = new MeshBasicMaterial();
+
+				} else {
+
+					c.material = new MeshStandardMaterial();
+					c.material.flatShading = true;
+
+				}
+
+				// if no debug rendering is happening then assign the material properties
+				if ( colorMode === NONE ) {
+
+					c.material.map = originalMaterial.map;
+					c.material.color.set( originalMaterial.map );
```

Comment:
TIL you can use a color instance with set now, should still be color though, will fixup in the other PR.

```suggestion
					c.material.color.set( originalMaterial.color );
```

Reply:
Thanks for the catch! PR'd too fast